src/styled.js

```diff
@@ -27,18 +33,25 @@ const styled = ({tagName, elementStyle, mountSheet}: StyledArgs) => {
   return class StyledElement extends Component {
     static tagName: string = tagName
     static style: ComponentStyleType = elementStyle
+    static contextTypes = {
+      [channel]: PropTypes.object
+    }
```

Comment:
and it also confusing part: theme is not dynamic part of the styles. its still static. so, when you debug style tags in the head, it gives you wrong sense of static/dynamic styles